main.js

```diff
@@ -0,0 +1,188 @@
+/*Heather Sleyster
+sleyster_a02b
+INFO 2134
+Thoendel
+03/17/2020
+*/
+window.addEventListener('load', function() {
+/*These statements assign the Form element to the form variable 
+and the select element named 'meal' to the select variable.
+*/
+const form = document.getElementById('contactForm');
+const meal = document.getElementByName('meal');
+
+/*The first statement assigns the elements with the name of mealOptionsHolder
+to the variable mealOption.This if statement checks the value of the 
+selected meal and if it is empty removes the 'visible' class from the form. 
+If breakfast is selected the visible class will be added to that option only, 
+if lunch is selected the visible class will be added to that option only,
+if dinner is selected the visible class will be added to that option.
+*/
+const mealOption = document.getElementsByName('mealOptionsHolder');
+    if (meal.value == 'breakfast'){
+        const breakfastOptions = getElementById('breakfastOptions');
+        breakfastOptions.classList.remove('hidden');
+        breakfastOptions.classList.add('visible');
+    }
+        else if (meal.value == 'lunch'){
+            const lunchOptions = getElementById('lunchOptions');
+            lunchOptions.classList.remove('hidden');
+            lunchOptions.classList.add('visible');
+        }
+            else if (meal.value == 'dinner') {
+                const dinnerOptions = getElementById('dinnerOptions');
+                dinnerOptions.classList.remove('hidden');
+                dinnerOptions.classList.add('visible');
+            }
+/*These statements assign the elements with id values of firstName and 
+lastName to the variables firstName and lastName, the age element with id 
+age to the age variable, the element with the id firstNameHolder to the
+firstNameHolder variable, the element with the id lastNameHolder to the 
+lastNameHolder variable, the element with id ageHolder to the ageHolder variable, 
+the element with the id emailAddress to the emailAddress variable, 
+and the element with id of mealRating to the variable mealRating.
+*/
+const firstName = document.getElementById('firstName');
+const lastName = document.getElementById('lastName');
+const age = document.getElementById('age');
+const firstNameHolder = document.getElementById('firstNameHolder');
+const lastNameHolder = document.getElementById('lastNameHolder');
+const ageHolder = document.getElementById('ageHolder');
+const emailAddress = document.getElementById('emailAdress');
+const mealRating = document.getElementById('mealRating');
+/*This is the event listener to listen for the blur event on the firstNameHolder
+element. It envokes the setNames function.
+*/
+firstNameHolder.addEventListener('blur', function() {
```

Comment:
we want the blur event on the input - not the div. So firstName.addEventListener.